src/XIVLauncher.Common/Game/IntegrityCheck.cs

```diff
@@ -108,6 +110,16 @@ private static IntegrityCheckResult DownloadIntegrityCheckForVersion(string game
             {
                 var relativePath = file.FullName.Substring(rootDirectory.Length);
 
+                // for unix compatibility with windows-generated integrity
+                if (PlatformHelpers.GetPlatform() == Platform.Win32 || PlatformHelpers.GetPlatform() ==  Platform.Win32OnLinux)
```

Comment:
Shouldn’t this be the opposite?

Reply:
Probably yes! Although...it ran. I'm gonna add some new debug checks and adjust accordingly. Will update the PR with a fix.

Reply:
Update: so when I copy my build from Windows and run it with .Net, GetPlatform() resolves to Win32, even on Linux. I'm assuming this is a compile-time thing, but that means it probably shouldn't be trusted as the source of truth. (So the code above was technically working fine for my testing, because it was wrong from the beginning)

Reply:
Thinking about it, it honestly doesn't need a platform check. We can just make the replacement always, as Windows doesn't really care about separators, but there isn't a good way to maintain compatibility with the old integrities while moving everything to `/`.  Kinda janky, but certainly not the worst possible way to handle this.